testnet/stacks-node/src/neon_node.rs

```diff
@@ -595,8 +595,9 @@ impl InitializedNeonNode {
             if let Some(key) = self.active_keys.pop() {
                 // sleep a little before building the anchor block, to give any broadcasted 
                 //   microblocks time to propagate.
-                debug!("Sleeping {} before issuing tenure", self.sleep_before_tenure);
-                thread::sleep(std::time::Duration::from_millis(self.sleep_before_tenure));
+                info!("Sleeping {} before issuing tenure", self.sleep_before_tenure);
+                let microblock_propagation_delay = 1500;
```

Comment:
Can this be a configuration-time option?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1648/commits/ab6a0cdf43b1293b2ab199f098ec3590c367569b